templates/manage_certs.py

```diff
@@ -32,7 +33,7 @@ def get_all_domains(config):
             line = line.strip()
             if line and not line.startswith("#"):
                 domain, backend = line.split(None, 1)
-                domains[domain] = backend
+                domains[domain.lower()] = backend
```

Comment:
@antoviaque I've modified the configuration management script to automatically convert all domain names to lower case for all new backend maps.  This is a better solution, since clients of the load balancer are no longer required to send lower case domain names.  This also makes it easy to test this in the integration test playbook.